src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -24,6 +24,7 @@
 
 #include "tvgCommon.h"
 #include "tvgArray.h"
+#include <string.h>
```

Comment:
According to your pr, we don't use str functions anymore, do we need this header?

Reply:
we are using functions from cstring in two places. 
One is the 'src/lib/tvgInitializer.cpp' and I changed a bit the code, so we don;t need to include 'cstring'
The other one is svg loader - in all files. also all files but 'tvgSvgUtil' have 'tvgSvgLoaderCommon' included. So I've included 'cstring' in the common file, and I've changed 'tvgSvgUtil' so it doesn't need 'cstring' any more

Reply:
@JSUYA but I included `string.h` no matter on what platform we compile the lib. I can't find any info why it's necessary only on windows...